docs/src/pages/de/migration/0.21.0.md

```diff
@@ -0,0 +1,227 @@
+
+```tsx
+// Div.tsx
+export default function Div(props) {
+  return <div />;
+}
+```
+
+In diesem Beispiel müsste auf `Div.tsx` mit `Div.tsx` verwiesen werden, nicht mit `Div.jsx`.
+
+```diff
+- import Div from './Div.jsx' // Astro v0.20
++ import Div from './Div.tsx' // Astro v0.21
+```
+
+Diese Änderung betrifft auch Style-Dateien.
+
+```scss
+// Div.scss
+div {
+  all: unset;
+}
+```
+
+```diff
+- <link rel="stylesheet" href={Astro.resolve('./Div.css')}>
++ <link rel="stylesheet" href={Astro.resolve('./Div.scss')}>
+```
+
+## Plugins
+
+In Astro v0.21 können Vite-Plugins innerhalb von `astro.config.mjs` konfiguriert werden.
+
+```js
+import { imagetools } from 'vite-imagetools';
+
+export default {
+  vite: {
+    plugins: [imagetools()],
+  },
+};
+```
+
+Um mehr über Vite-Plugins zu lernen, lies bitte Vites [Plugin Guide](https://vitejs.dev/guide/using-plugins.html).
+
+## Benutzerdefinierte Renderer
+
+In Astro v0.21 sollten Plugins ab jetzt `viteConfig()` verwenden.
+
+```diff
+// renderer-svelte/index.js
++ import { svelte } from '@sveltejs/vite-plugin-svelte';
+
+export default {
+  name: '@astrojs/renderer-svelte',
+  client: './client.js',
+  server: './server.js',
+-  snowpackPlugin: '@snowpack/plugin-svelte',
+-  snowpackPluginOptions: { compilerOptions: { hydratable: true } },
++  viteConfig() {
++    return {
++      optimizeDeps: {
++        include: ['@astrojs/renderer-svelte/client.js', 'svelte', 'svelte/internal'],
++        exclude: ['@astrojs/renderer-svelte/server.js'],
++      },
++      plugins: [
++        svelte({
++          emitCss: true,
++          compilerOptions: { hydratable: true },
++        }),
++      ],
++    };
++  },
+}
+```
+
+Um mehr über Vite-Plugins zu lernen, lies bitte Vites [Plugin Guide](https://vitejs.dev/guide/using-plugins.html). 
+
+> In früheren Versionen wurden diese Plugins mittels `snowpackPlugin` oder `snowpackPluginOptions` konfiguriert.
+
+## Änderungen bei der Gestaltung
+
+### Autoprefixer
+
+Autoprefixer wird nicht mehr automatisch eingebunden. Aktiviere Autoprefixer in zwei Schritten:
+
+1. Installiere zunächst die aktuelle Version (`npm i autoprefixer`)
+2. Erzeuge die Datei `postcss.config.cjs` im Wurzelverzeichnis deines Projekts mit:
+   ```js
+   module.exports = {
+     plugins: {
+       autoprefixer: {},
+     },
+   };
+   ```
+
+### Tailwind-CSS
+
+Stelle sicher, dass Post-CSS installiert ist. Dies war optional in vorherigen Versionen, ist aber nun Voraussetzung:
+
+1. Intalliere die aktuelle Version von Post-CSS (`npm i -D postcss`)
+2. Erzeuge die Datei `postcss.config.cjs` im Wurzelverzeichnis deines Projekts mit:
+   ```js
+   module.exports = {
+     plugins: {
+       tailwindcss: {},
+     },
+   };
+   ```
+   Für weitere Informationen lies die [Tailwind CSS Documentation](https://tailwindcss.com/docs/installation#add-tailwind-as-a-post-css-plugin)
+
+[snowpack]: https://www.snowpack.dev
```

Comment:
Do we still need the link to snowpack? If yes, I'd prefer it being the last link